modules/detectron/ps_roi_pool_op.cu

```diff
@@ -123,8 +123,13 @@ __global__ void PSRoIPoolForward(
       roundf(offset_bottom_rois[4]) + 1.) * spatial_scale;
 
     // Force too small ROIs to be 1x1
+    #ifdef __HIP_PLATFORM_HCC__
+    T roi_width = fmax(roi_end_w - roi_start_w, 0.1);  // avoid 0
```

Comment:
use `c10::cuda::compat::max` instead, which has the right dispatch to `fmax` and `fmaxf` based on the instantiated template type. (and avoid ifdef)